src/app/login/githublogin.tsx

```diff
@@ -32,11 +47,21 @@ const GithubLogin: React.FC = () => {
                 icon={<GithubIcon />}
                 iconPosition="left"
                 size="lg"
-                style={{ backgroundColor: 'black' }}
+                style={{ backgroundColor: 'black', marginRight: '20px' }}
                 onClick={handleGitHubLogin}
               >
                 Sign in with GitHub
               </Button>
+              <Button
+                variant="primary"
+                icon={<CustomLogoIcon />}
+                iconPosition="left"
+                size="lg"
+                style={{ backgroundColor: 'black' }}
```

Comment:
```suggestion
                style={{ backgroundColor: '#24274e' }}
```

Quick hack to make this slightly less primary-looking (only required once).